### PR TITLE
Support emacs style key binding 

### PR DESCRIPTION
**Pull request by storyn26383**
With this PR, we can use `CTRL_P`, `CTRL_F` `CTRL_N`, `CTRL_B` and `CTRL_H` to do `UP`,  `RIGHT`, `DOWN`, `LEFT` and `BACKSPACE`.

### CODE REVIEW

File: src/MultiSelectPrompt.php [hunk 1]
```diff
@@ -62,8 +62,8 @@ public function __construct(
         $this->reduceScrollingToFitTerminal();
 
         $this->on('key', fn ($key) => match ($key) {
-            Key::UP, Key::UP_ARROW, Key::LEFT, Key::LEFT_ARROW, Key::SHIFT_TAB, 'k', 'h' => $this->highlightPrevious(),
-            Key::DOWN, Key::DOWN_ARROW, Key::RIGHT, Key::RIGHT_ARROW, Key::TAB, 'j', 'l' => $this->highlightNext(),
+            Key::UP, Key::UP_ARROW, Key::LEFT, Key::LEFT_ARROW, Key::SHIFT_TAB, Key::CTRL_P, 'k', 'h' => $this->highlightPrevious(),
+            Key::DOWN, Key::DOWN_ARROW, Key::RIGHT, Key::RIGHT_ARROW, Key::TAB, Key::CTRL_N, 'j', 'l' => $this->highlightNext(),
```
Comment (jessarcher) [inline src/MultiSelectPrompt.php:66]: Should this have `CTRL_F` and `CTRL_B` for consistency with the existing left/right/h/l behaviour?

File: src/SelectPrompt.php [hunk 2]
```diff
@@ -70,8 +70,8 @@ public function __construct(
         }
 
         $this->on('key', fn ($key) => match ($key) {
-            Key::UP, Key::UP_ARROW, Key::LEFT, Key::LEFT_ARROW, Key::SHIFT_TAB, 'k', 'h' => $this->highlightPrevious(),
-            Key::DOWN, Key::DOWN_ARROW, Key::RIGHT, Key::RIGHT_ARROW, Key::TAB, 'j', 'l' => $this->highlightNext(),
+            Key::UP, Key::UP_ARROW, Key::LEFT, Key::LEFT_ARROW, Key::SHIFT_TAB, Key::CTRL_P, 'k', 'h' => $this->highlightPrevious(),
+            Key::DOWN, Key::DOWN_ARROW, Key::RIGHT, Key::RIGHT_ARROW, Key::TAB, Key::CTRL_N, 'j', 'l' => $this->highlightNext(),
```
Comment (jessarcher) [inline src/SelectPrompt.php:74]: Same here re: `CTRL_F` and `CTRL_B`

**Review by jessarcher — COMMENTED** [review]
Very cool!

Makes me wonder whether we should add some readline keys like `CTRL_A` and `CTRL_E`, as well as `Home` and `End`. They'd probably need to work differently depending on whether you're currently focused on a list or a typed input. Someone can PR that separately if you don't feel like it belongs here though.

File: src/MultiSelectPrompt.php (hunk 1 above)
Comment (storyn26383) [reply src/MultiSelectPrompt.php:66]: Sure! I lost it, sorry about that.

**Comment by storyn26383** [comment]
Good idea!
We can add home and end feature in next RP 🎉

**Pull request merged**